lang/cs/Org.Apache.REEF.Network/packages.config

```diff
@@ -18,10 +18,49 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Extensions.Caching.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Caching.Memory" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Options" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Primitives" version="1.1.0" targetFramework="net451" />
```

Comment:
> Sorry I should be more concrete, the commit message I added when pushing this code to the tip of my branch follows the exact format specified by you and Doug, am trying to understand where the disconnect actually lies

That is odd. The commit as shown in this pull request does not have that message. 

When you send a PR, it is generally a good idea to minimize the diff to current `master` and then squash all commits to one and rebase it onto current `master`. That one commit should have the right commit message, and subsequent commits are made to that branch in response to the review. 

This process has worked well for me in the past, and minimizes the code that needs review in the pull request.